tests/integ/sagemaker/workflow/test_workflow.py

```diff
@@ -0,0 +1,1010 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
so, in the future, what tests should be added to this `test_workflow.py` ?

Reply:
As for adding new features, which is specifically testing the new feature e.g. the step type, we should add a separate test file for it.
As for the test which is not specific to any step type but is more on the entire workflow and on how each step works together, we can add such tests in `test_workflow.py`